readme.md

```diff
@@ -93,6 +93,18 @@ Respect ignore patterns in `.gitignore` files that apply to the globbed files.
 
 Returns `string[]` of matching paths.
 
+### globby.stream(patterns, options?)
+
+Returns a [`stream.Readable`](https://nodejs.org/api/stream.html#stream_readable_streams) of matching paths.
+
+Since Node.js 10, [readable streams are iterable](https://nodejs.org/api/stream.html#stream_readable_symbol_asynciterator), so you can loop over glob matches in a [`for await...of` loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of) like this:
```

Comment:
This an the below example should be in the TS definition too.

Reply:
Do you mean in the docs for the `stream` method around here
https://github.com/sindresorhus/globby/blob/6618d820fa6bd0580dae6d675984dbd4cfe5ab4a/index.d.ts#L117

, or in the `index.test-d.ts`? I've already added a loop like this there:
https://github.com/sindresorhus/globby/blob/6618d820fa6bd0580dae6d675984dbd4cfe5ab4a/index.test-d.ts#L68

Reply:
> Do you mean in the docs for the stream method around here

Yes